firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+TabToolbarDelegate.swift

```diff
@@ -49,10 +49,11 @@ extension BrowserViewController: TabToolbarDelegate, PhotonActionSheetProtocol {
         } else {
             navigationHintDoubleTapTimer = nil
             let action = ToolbarAction(windowUUID: windowUUID, actionType: ToolbarActionType.navigationButtonDoubleTapped)
-            store.dispatchLegacy(action)
+            store.dispatch(action)
         }
     }
 
+    // This is never called. Laurie - ask why
```

Comment:
@thatswinnie @Foxbolts you two might be the ones having the most context on this `configureNavigationContextualHint` according to git blame haha. This isn't called, can we remove it that contextual hint?

Reply:
Wth, why was my Xcode not telling me this 😆